pkg/services/sqlstore/searchstore/search_test.go

```diff
@@ -179,6 +179,8 @@ func createDashboards(t *testing.T, store db.DB, startID, endID int, orgID int64
 			dash.UID = util.GenerateShortUID()
 			dash.CreatedBy = 1
 			dash.UpdatedBy = 1
+			dash.Created = time.Now()
```

Comment:
required because the `NewDashboardFromJson` model previously set these based on the value in "id" 🤷